ui/apps/platform/src/services/ComplianceEnhancedService.ts

```diff
@@ -338,24 +338,25 @@ export function getSingleClusterResultsByScanConfig(
     }
     const searchQuery = getRequestQueryStringForSearchFilter({
         'Cluster ID': clusterId,
-        'Compliance Scan Config Name': scanName,
         ...searchFilter,
     });
 
     let offset: number | undefined;
     if (typeof page === 'number' && typeof pageSize === 'number') {
         offset = page > 0 ? page * pageSize : 0;
     }
-    const query = {
-        query: searchQuery,
-        pagination: { offset, limit: pageSize, sortOption },
+    const queryParameters = {
+        query: {
+            query: searchQuery,
+            pagination: { offset, limit: pageSize, sortOption },
+        },
     };
-    const params = qs.stringify(query, { arrayFormat: 'repeat', allowDots: true });
+    const params = qs.stringify(queryParameters, { arrayFormat: 'repeat', allowDots: true });
 
     return axios
         .get<{
             scanResults: ComplianceScanResult[];
-        }>(`${complianceResultsServiceUrl}/results?${params}`)
+        }>(`${complianceResultsServiceUrl}/results/${scanName}?${params}`)
```

Comment:
Remind me, is response from another request the source of `scanName` or does it ever come directly from UI input? If ever the latter, is there validation to ensure safety to interpolate into the endpoint address?

Reply:
Yes, the `scanName` is being passed around from `/v2/compliance/results/stats/cluster`